changelog.d/20231115_110934_andrey_update_helm.md

```diff
@@ -0,0 +1,7 @@
+### Changed
+
+- \[Helm\] Minimum compatible Kubernetes version is 1.19.0.
+  (<https://github.com/opencv/cvat/pull/7132>)
+
+- \[Helm\] The CVAT hostname can be configured with `ingress.hostname` option.
```

Comment:
This isn't the only change. You've also removed `ingress.hosts` and `ingress.tls`.

Reply:
I'm sure that the tls options never worked, but ok I'll add the entry